types/src/error.rs

```diff
@@ -16,12 +16,36 @@ impl<T: fmt::Display> fmt::Display for Mismatch<T> {
 	}
 }
 
+/// Invalid params.
+#[derive(Debug)]
+pub struct InvalidParams;
```

Comment:
It was just that `register_method_without_context` can only fail when the params couldn't be parsed so it was introduced to avoid matching on unreachable enum variants

Reply:
I think it could be removed, lemme check